presto-spi/src/main/java/com/facebook/presto/spi/statistics/PlanStatistics.java

```diff
@@ -80,18 +77,13 @@ public double getConfidence()
     }
 
     @JsonProperty
-    @ThriftField(4)
-    public Estimate getNullJoinBuildKeyCount()
+    @ThriftField(value = 6, requiredness = OPTIONAL)
```

Comment:
Tested locally that it is backward compatible

Reply:
https://github.com/prestodb/presto/blob/master/redis-hbo-provider/src/test/java/com/facebook/presto/statistic/TestHistoricalStatisticsSerde.java#L32 
Nit: You can also add extend the serde tests here